BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -241,9 +247,41 @@ final class LocationView: UIView,
         } else {
             updateUIForLockIconDisplay()
         }
+        animateIconAppearance(animated: isAnimationEnabled)
         urlTextFieldTrailingConstraint?.constant = -UX.horizontalSpace
     }
 
+    private func animateIconAppearance(animated: Bool) {
```

Comment:
Wouldn't it make sense to check `UIAccessibility.isReduceMotionEnabled` directly in this method instead of passing `animated`?